web/tests/firestore/firestore.test.js

```diff
@@ -0,0 +1,22 @@
+import { auth } from '../../firebase';
+
+let user;
```

Comment:
Sharing resources like this between tests is pretty bad practice - tests should be completely independent of one another, so as to avoid unexpected interference when a test doesn't clean back properly after itself. This might work if cleanup is done with `afterEach` clauses, but that would still be pretty messy.

It might feel redundant for each test to create and remove a user each time, but isolation > chasing down bugs in the test suite 😢

Reply:
Yeah I do agree this is a pretty shitty test suite in many ways, but the intention was to quickly prototype/experiment with Firebase authentication system. Like I’m not even sure if we’re going to use this test suite for actual firestore security rules testing, so I just wanted to do the minimum work required to validate some Firebase authentication methods. Is it okay if we implement this test suite the correct way once we get to a point where we’re actually building more tests on top of this? I could add a comment about it for now. If you feel strongly about implementing it now, then I could also do that since it’s pretty low effort

Reply:
In that case a comment will do for the time being I think! Thanks 👍